content/applications/sales/sales/products_prices/products/variants.rst

```diff
@@ -1,225 +1,178 @@
+   to know what they're selecting.
 
-Before you can use product variants, you must first activate the product
-variants in the settings. To do so, you must go to the Sales app. In the
-menu :menuselection:`Configuration --> Settings`, locate the **Products Variants** line,
-and tick the option **Products can have several attributes**, then click
-on **Apply**.
+A color code can be selected either by dragging the slider around or by entering a specific HTML
+color code (e.g. #FF0077).
 
-.. image:: variants/variant01.png
+.. image:: variants/picking-a-color.png
    :align: center
+   :alt: Selecting a color.
 
-Creating products with variants
--------------------------------
+.. tip::
+   Attributes can also be created directly from the product template by adding a new line and
+   typing the name into the :guilabel:`Variants` tab.
 
-Once you have activated the variant option, you can add variants to your
-products. To do so, go to the Sales module, :menuselection:`Sales --> Products`.
-It is also accessible from the Purchase and inventory modules.
+After an attribute is added to a product, a new tab appears on the attribute's page called
+:guilabel:`Related Products`. This tab lists every product in the database that is currently using
+the attribute.
 
-Now, click on the product you wish to add variants to.
+Creating product variants
+-------------------------
 
-In the product page, a new tab called Variants has appeared. The number
-in purple written on top is the number of variants this product
-currently has. To add new variants, click on the tile. In the new
-window, click on **Create**.
+Head over to the :guilabel:`Products` page by clicking :menuselection:`Sales --> Products -->
+Products`, then either select an existing product or create a new one by tapping
+:guilabel:`Create`.
 
-In **Attributes**, click on the rolldown menu and select the type of
-variance you wish to add. If the variant does not yet exist, you can
-create it on the fly by clicking on Create and edit…
+The smart button at the top of the product template indicates the number of currently configured
+variants on this product.
 
-.. image:: variants/variant02.png
+.. image:: variants/variant-smart-button.png
    :align: center
+   :alt: Variants smart button.
 
-In the Attributes window, the **Value** field is the description of the
-attribute such as Green, Plastic or 32GB. The **Attribute** field is the
-type of variant such as Color, Material or Memory.
-
-.. image:: variants/variant03.png
-   :align: center
-
-You can add a cost for the variant on the fly by adding it in the
-**Attribute Price Extra** field, or choose to modify it later. Click on
-**Save**.
-
-You can also add a different barcode and internal reference to the
-variant.
+To add a new variant, click on the :guilabel:`Variants` tab, then click on :guilabel:`Add a line`
+to add any attributes and values.
```

Comment:
Add `` When all the attributes and values have been added, click :guilabel:`Save`. `` to the end of this paragraph